### ISSUE TITLE
Confusion Matrix

### ISSUE THREAD

**Issue by MalihaUCF** [King-Of-Knights/Keras-ACGAN-CIFAR10#4]
Can you please share how u made the confusion matrix?

**Comment by King-Of-Knights** [comment]
[Here](http://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html#sphx-glr-auto-examples-model-selection-plot-confusion-matrix-pyis) is what you want

**Comment by figuring-gan** [comment]
Thanks! what changes did you make to get that link working with your code ?

**Comment by King-Of-Knights** [comment]
@figuring-gan First, you should use D to predict Cifar-10 test dataset and get the predict label and true label. Then, it won't be difficult to plot Confusion Matrix by feed predict label and true label into the code I mentioned. Please have a try!
